src/EnergyExporter/Options/ExportOptions.cs

```diff
@@ -26,4 +29,20 @@ public class InfluxDbOptions
         [Required]
         public string Token { get; init; } = null!;
     }
+
+    public class MqttOptions
+    {
+        [Required]
+        public string TcpServer { get; init; } = null!;
+
+        public int Port { get; init; } = 1883;
+
+        public string ClientId { get; init; } = "EnergyExplorer";
+        
+        public string Topic { get; init; } = "EnergyExplorer";
```

Comment:
You probably meant `EnergyExporter`?

Reply:
Whoops :-)   - sorry, that wasn't on purpose. Corrected.